core/primitives/src/views.rs

```diff
@@ -378,6 +378,14 @@ impl From<Tip> for BlockStatusView {
     }
 }
 
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
+#[derive(Serialize, Deserialize, Debug)]
+pub struct EpochInfoView {
+    pub epoch_id: CryptoHash,
+    pub start_time: String,
```

Comment:
what is the timezone? UTC?

Reply:
I believe this one includes the timezone. Check out the string in the screenshot!